### ISSUE TITLE
Performance on nuscenes test

### ISSUE THREAD

**Issue by aurorasyu** [mit-han-lab/bevfusion#242]
Hi, thanks for your excellent work.

I have some questions about the result on nusenes test set. Our result of test set trained from train set is similar to the result from train+val set. But you result of the test is 2% mAP heigher when train on train+val. Is there anything different when train on train+val.

Thanks for your help.

**Comment by sandropapais** [comment]
I have a similar issue. When I evaluate the pretrained model provided the performance is mAP=0.6846 and NDS=0.7143 (same as the paper). However, when I train the model from scratch using the provided config and evaluate it, the performance is mAP=0.6440 and NDS=0.6933. So there is a gap of 4% mAP and 2% NDS when I try to reproduce the results.

Is there a different config file or an older version that should be used to reproduce the results?

**Comment by kentang-mit** [comment]
We use the same schedule for models on val and models on test. The only difference is that we use the training set for models evaluated on the validation set and we use train+val set for models evaluated on the test set. As far as I know, CenterPoint, TransFusion and our method all have slightly higher test mAP (compared with val mAP), but the performance difference varies from model to model.

**Comment by aurorasyu** [comment]
The Lidar pretrain model also use train+val? or only fusion use train+val?
when i use the train+val to train the lidar model, I find the performance on truck has twenty points lower campare to train on train set.

**Comment by aurorasyu** [comment]
Thanks for your answer! 
Another question is that the train+val you used is combine the train.pkl+val.pkl during create data?

**Comment by kentang-mit** [comment]
@aurorasyu, Yes, you are right.

**Comment by kentang-mit** [comment]
Closed due to inactivity.
